### DISCUSSION TITLE
MW4 2nd use

### DISCUSSION THREAD

**Discussion by alanrock24** [mworion/MountWizzard4#272] [Q&A]
I tried again tonight and got basically the same problems, although this time the mount DID actually hit all the points in the model.
When it finished I got the same errors as last night, except I had a slightly lower RMS-- about 6500".
Any ideas?
Getting ready to give up!!
[mw4-2022-08-28.log](https://github.com/mworion/MountWizzard4/files/9439082/mw4-2022-08-28.log)

**Comment by alanrock24** [comment]
![MW4pages](https://user-images.githubusercontent.com/103699926/187057519-0d27a0b0-c0c6-46a9-a41a-7baa1df6b05f.jpg)

**Reply by mworion** [reply]
Hi Alan,
sorry to hear that also the last build did not succeed. Right now I'm on a trip till next weekend. So there is no chance for me to check in my environment in comparison. The image you posted looks so far good. If you don't mind we'll do an online session after I'm back. But anyway: if you are satisfied with the results of NINA or perhaps ModelCreator, you are able to work. 
If you want to move on with MW4, please enable debug logging (under settings/log). Enable keep images from model run (under imaging) and remove build points close to meridian (under model points).
You also relax timing to conservative under model build if you see star trails in the images during model build.
From my experience there might be a single point who makes the big error. You could see that under manage model as red point. 
Michel
